training/philadelphia-pediatric/crop-composed-to-top-images-and-labels.py

```diff
@@ -0,0 +1,220 @@
+"""
+This script is used to crop the composed images and labels to the same dimensions as the top images and then
+convert the BIDS datastructure to nnU-Net datastructure.
+
+This script is necessary because we need to crop composed images and labels to top images size to be able to use them
```

Comment:
```suggestion
This script is necessary because we need to crop composed images and labels to top images (covering C spine) to be able to use them
```